app.js

```diff
@@ -0,0 +1,20 @@
+let express = require('express');
+let app = express();
+let bodyParser = require('body-parser');
+
+import { parsedCookiesMW, parsedQueryMW } from './middlewares';
+
+import productRoutes from './routes/product.route';
```

Comment:
Consider following the same approach as you did above with "middlewares" folder and importing an object destructuring all the modules you need along the way.
Also, consider removing ".route" from the name - take a look at the above explanation about Node.js ecosystem module naming.

`import { productRoutes, userRoutes } from './routes'; `